### ISSUE TITLE
Consider using 

### ISSUE THREAD

**Issue by ssbarnea** [ansible-community/awesome-ansible#86]
Take a look at similar awesome listing that uses automation for listing popular and trending items: https://github.com/pawamoy/best-of-mkdocs

Maybe we should make use of the same solution to maintain our listing?

**Comment by KeyboardInterrupt** [comment]
Hi, sorry for the slow response!
Appreciate the suggestion! I'm a big fan of automation, but in this specific instance, automatically gathering and adding everything related to Ansible might not be advantageous for the List. Its primary aim is to be community-driven and maintained, ensuring it remains a resource built by the community for the community.

Still, I like the Idea and dont want to dismiss it right away, and I will take a look at the mentioned repository and see if there is something there!

^C

**Comment by samccann** [comment]
yes I'm inclined to agree. We decided 'somewhere' (sorry lost track) that we should be using this repo instead of adding individual tools to the documentation page. So we want to be sure the tools we have here are valid/worthwhile/maintained etc. Hard to do that with automation.
